go.mod

```diff
@@ -5,16 +5,18 @@ go 1.18
 require (
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hashicorp/hcl v1.0.0
+	github.com/maxbrunsfeld/counterfeiter/v6 v6.5.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/spf13/cobra v1.4.0
+	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.10.1
-	gotest.tools/v3 v3.1.0
+	github.com/spf13/viper v1.13.0
+	gotest.tools/v3 v3.3.0
 	k8s.io/api v0.25.4
 	k8s.io/apiextensions-apiserver v0.25.4
 	k8s.io/apimachinery v0.25.4
 	k8s.io/client-go v0.25.4
 	k8s.io/code-generator v0.25.4
+	knative.dev/client v0.35.1
```

Comment:
That can't happen actually. Because we'll create a cyclic dependency in kantive/client eventually.

Reply:
yes, this got changed due to the rebase. I am fixing it.